app/models/concerns/progress.rb

```diff
@@ -0,0 +1,17 @@
+module Progress
```

Comment:
Just wondering: why not using an abstract class here? It makes sense to think `Indicador` and `Progress` as subclasses of `Progress`.

I think that choice would be consistent with that in `Content` class: https://github.com/mumuki/mumuki-domain/blob/master/app/models/content.rb

Reply:
> `Indicador` and `Progress` as subclasses of `Progress`

Guessing you mean `Indicator` and `Assignment` as subclasses of `Progress` ? Honestly, it didn't occur to me :stuck_out_tongue_closed_eyes: I'll go for that

Reply:
:rofl: